### ISSUE TITLE
Notify user about image adjustments/normalization

### ISSUE THREAD

**Issue by AnneCarpenter** [CellProfiler/CellProfiler#3448]
The discussion in #3396  is one of those simple but super important things! 

I've always been a bit uneasy about the fact that some kind of adjustment is usually necessary, yet we don't want it to be happening without the user realizing.

Is it practical to add some kind of indicator somewhere? For example adding "(stretched)" or "log-normalized" next to the image title above each image within a window?

**Comment by bethac07** [comment]
I'm reasonably sure you'd have to change the code in every module individually to do so.  What is your particular concern about it happening without the user's knowledge?  Most pieces of software do this- ImageJ notably among them.

Would you be more comfortable if CP's default behavior was raw/unstretched rather than normalized?  That'd be easy enough to change, but it then puts an extra step on the user to go a) figure out that there's a contrast stretch default option and b) figure out where it is and how to change it.

**Comment by AnneCarpenter** [comment]
None of the options are awesome. You are exactly right that the default being unstretched avoids any chance of misleading people but is almost always useless for viewing images. 

So I like the current default being stretched, I just wish it was more obvious that this had been done. Maybe there is another location/way to make this apparent that is easier and less cluttery.

My concern is when people open up two images side by side and can't figure out why one looks unexpectedly brighter than the other.

**Comment by mcquin** [comment]
What if we added the normalization method to the pixel intensity bottom bar?

<img width="898" alt="screen shot 2018-01-09 at 10 25 37 am" src="https://user-images.githubusercontent.com/4721755/34728267-762a6eae-f527-11e7-855d-c2039f859e44.png">

**Comment by AetherUnbound** [comment]
I'm all for that idea!

**Comment by bethac07** [comment]
I like it in theory; is the plan that when you hover over any given sub-panel it'll tell you the normalization?  I'd think based on the fact that different subplots can be normalized differently you'd have to do it that way.

**Comment by mcquin** [comment]
> is the plan that when you hover over any given sub-panel it'll tell you the normalization?

Yeah. I think it's possible that we could always have the normalization method displayed (you wouldn't have to hover over the panel first).

**Comment by bethac07** [comment]
>Yeah. I think it's possible that we could always have the normalization method displayed (you wouldn't have to hover over the panel first).

I'm fine with that too, but if we do that, we should change the text to "Default intensity normalization: Normalized", because once you start interacting with the subplots you can give them all different intensity normalization modes.

**Comment by mcquin** [comment]
I see. We can make it such that if the subplots have different normalizations, then the text updates to show the selected subplot's normalization when the mouse hovers over it. How does that sound?

**Comment by AnneCarpenter** [comment]
@bethac07: how important is it that subplots can be normalized independently of each other? Because maybe THAT is what we ditch here so that we can add a single intensity normalization in the bar as per @mcquin's helpful suggestion. 

In such a case, a single set of radio buttons to try different normalizations would be handy (but I suspect @bethac07 will say that you rarely want to futz with the normalization in practice so it's too much real estate/clutter to devote to something mostly unused.)

**Comment by bethac07** [comment]
@AnneCarpenter it's important to be able to tweak them independently.  

I'm totally fine with any of the 3:
* Show only once you hover (as we do with X, Y, Intensity)
* Show only what the default was, you're responsible for keeping track of how you've played with things in the module (fine in 99% of cases I'd wager)
* Show the default on startup then switch behaviors after hovering.

Whatever is easiest and whatever other people like.

**Comment by AnneCarpenter** [comment]
I'd vote for option 1, hovering, if the implementation pain is equal. A nice side benefit is it sort of catches your attention.
